dev-python/b2sdk/b2sdk-1.2.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
+inherit distutils-r1
+
+DESCRIPTION="The client library for BackBlaze's B2 product"
+HOMEPAGE="https://github.com/Backblaze/b2-sdk-python"
+SRC_URI="https://github.com/Backblaze/b2-sdk-python/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+PATCHES=(
+	"${FILESDIR}/${P}_skip-integration-test.patch"
+	"${FILESDIR}/${P}_skip-license-header-test.patch"
```

Comment:
- ${P}_ looks odd, usually we do ${P}-...
- I usually do ${P} if it's an upstreamed patch or ${PN}-versionItIsBasedOn if it's not. This is just a style/hint thing though.

Reply:
Good to know I'll make this change (on my other ebuild submission too)